openml/utils.py

```diff
@@ -40,6 +42,53 @@ def extract_xml_tags(xml_tag_name, node, allow_none=True):
         else:
             raise ValueError("Could not find tag '%s' in node '%s'" %
                              (xml_tag_name, str(node)))
+
+
+def _tag_entity(entity_type, entity_id, tag, untag=False):
+    """Abstract function that can be used as a partial for tagging entities
```

Comment:
I do not see where this is used as a `partial` (assuming you mean `functools.partial`). Could you please clarify the docstring?

Reply:
I haven't put these functions in yet. I will change the description, as this function can also be used standalone.